crates/commitlog/src/repo/mod.rs

```diff
@@ -17,6 +18,23 @@ pub use fs::Fs;
 #[cfg(test)]
 pub use mem::Memory;
 
+#[derive(Debug, Default, Clone, Copy)]
+pub struct TxOffset(u64);
```

Comment:
Made it alias, newtype is not useful right now but may be when we use different Indexes like timestamps based.
Though, It will require bunch of mechanical change, can be done separately or when needed.